Cask

```diff
@@ -9,4 +9,5 @@
  (depends-on "ert-runner")
  (depends-on "package-lint")
  (depends-on "annalist")
- (depends-on "magit"))
+ (depends-on "magit")
+ (depends-on "mpdel"))
```

Comment:
Probably don't need this unless you're adding tests.

Reply:
No new test is added, but the ci is failed at first time opening the pr without these lines.

Reply:
I pushed a new commit without this, and the ci failed.